src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -302,11 +303,6 @@ export default class ChangeBusinessType extends Mixins(CommonMixin) {
     return this.getResource.changeData?.typeChangeInfo
   }
 
-  /** Whether to show the Change Info tooltip (firm and coop filings only). */
-  get showChangeInfoTooltip ():boolean {
-    return (this.isCorpClassFirm || this.isCoop)
-  }
```

Comment:
Logic moved into template, since it's simple and only used once.

(Also, while a getter is cached, which can improve performance -- lots of re-rendering -- in this scenario the type of filing and business won't change at all during this session.)